<?xml version="1.0" ?>
<launch>
  <rosparam command="load" file="$(find NRS_FT_AQ)/config/VR_config.yaml"/>
  <node name="FT_AQ" pkg="NRS_FT_AQ" type="FT_AQ_main" output="screen"/>
</launch>